hooks/useNWACObservations.ts

```diff
@@ -7,47 +7,51 @@ import * as Sentry from 'sentry-expo';
 
 import {Logger} from 'browser-bunyan';
 import {ClientContext, ClientProps} from 'clientContext';
-import {add, formatDistanceToNowStrict} from 'date-fns';
+import {add, formatDistanceToNowStrict, sub} from 'date-fns';
```

Comment:
changes in this file look big, but it's just making the fetch strategy mirror the NAC strategy - fetch two week windows at a time.

using `limit` and `offset` over a 1 year period produces really slow performance, so fetching a series of smaller time windows works better